sdk/identity/azure-identity/pom.xml

```diff
@@ -139,6 +139,20 @@
         </configuration>
       </plugin>
 
+      <plugin>
+        <groupId>com.diffplug.spotless</groupId>
+        <artifactId>spotless-maven-plugin</artifactId>
+        <version>2.30.0</version> <!-- {x-version-update;com.diffplug.spotless:spotless-maven-plugin;external_dependency} -->
+        <configuration>
+          <java>
+            <excludes>
+              <exclude>**/PowerShellUtil.java</exclude>
```

Comment:
Use `spotless:off` and `spotless:on` instead of excluding the whole file - https://github.com/diffplug/spotless/tree/main/plugin-gradle#spotlessoff-and-spotlesson